### PR TITLE
Coloured output for init.sh

### PR DESCRIPTION
**Pull request by boonya**

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -11,40 +22,40 @@ filetype off
...
-"-------------------=== Python  ===-----------------------------
-Plugin 'klen/python-mode'                   " Python mode (docs, refactor, lints...)
-Plugin 'davidhalter/jedi-vim'               " Jedi-vim autocomplete plugin
-Plugin 'mitsuhiko/vim-jinja'                " Jinja support for vim
-Plugin 'mitsuhiko/vim-python-combined'      " Combined Python 2/3 for Vim
-Plugin 'hynek/vim-python-pep8-indent'       " PEP8 indent
-Plugin 'jmcantrell/vim-virtualenv'          " Virtualenv support in VIM
+    Plugin 'VundleVim/Vundle.vim'               " let Vundle manage Vundle, required
+
+    "-------------------=== Code/Project navigation ===-------------
```
Comment (rmk135) [inline vimrc:27]: There are some tabs in this section between code and comment, can you please remove them?

File: vimrc [hunk 2]
```diff
@@ -11,40 +22,40 @@ filetype off
...
+    Plugin 'scrooloose/nerdtree' 	    	    " Project and file navigation
+    Plugin 'majutsushi/tagbar'          	    " Class/module browser
+    Plugin 'kien/ctrlp.vim'                     " Fast transitions on project files
+
+    "-------------------=== Other ===-------------------------------
```
Comment (rmk135) [inline vimrc:32]: The same problem with tabs in this one

File: vimrc (hunk 1 above)
Comment (boonya) [inline vimrc:27]: What about now?

Comment (rmk135) [inline vimrc:27]: Awesome! Thanks!

**Pull request merged**